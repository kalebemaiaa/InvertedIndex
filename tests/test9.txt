QueryOr(QueryWord("senador"), QueryAndS(QueryWord("deputado"), QueryWord("assessor")))
5643.raw
1803.raw
2349.raw
4465.raw
4980.raw
2786.raw
4663.raw
11262.raw
6165.raw
2274.raw
617.raw
4845.raw
6220.raw
3796.raw
2657.raw
957.raw
2187.raw
4491.raw
5172.raw
6429.raw
5478.raw
3454.raw
6317.raw
3850.raw
10936.raw
4422.raw
11175.raw
1578.raw
4523.raw
4919.raw
8729.raw
1971.raw
1523.raw
5380.raw
11339.raw
3055.raw
11400.raw
5484.raw
4542.raw
5062.raw
6319.raw
11416.raw
3126.raw
11911.raw
4990.raw
206.raw
759.raw
10884.raw
11009.raw
2243.raw
5755.raw
11084.raw
2014.raw
2496.raw
2617.raw
3030.raw
11755.raw
1649.raw
3778.raw
2893.raw
4034.raw
490.raw
5728.raw
12016.raw
1194.raw
11979.raw
11252.raw
4361.raw
6305.raw
2675.raw
4439.raw
491.raw
1623.raw
3526.raw
820.raw
4165.raw
11737.raw
1559.raw
3256.raw
4573.raw
1720.raw
3244.raw
1731.raw
479.raw
1067.raw
726.raw
5817.raw
3287.raw
742.raw
1973.raw
4869.raw
3891.raw
11849.raw
11215.raw
3111.raw
90.raw
240.raw
4199.raw
2057.raw
4581.raw
2983.raw
4963.raw
1692.raw
3224.raw
6117.raw
2079.raw
1311.raw
2691.raw
6177.raw
3419.raw
11461.raw
1597.raw
2667.raw
4080.raw
25.raw
4100.raw
2032.raw
5028.raw
2028.raw
1083.raw
6003.raw
4725.raw
2773.raw
1874.raw
1058.raw
6263.raw
493.raw
5617.raw
5365.raw
11031.raw
804.raw
4339.raw
5369.raw
352.raw
6024.raw
764.raw
755.raw
1962.raw
5599.raw
5775.raw
1543.raw
4344.raw
5798.raw
2450.raw
1461.raw
4564.raw
1245.raw
4448.raw
1697.raw
4870.raw
5449.raw
5189.raw
5537.raw
4732.raw
4893.raw
6066.raw
1169.raw
4501.raw
11391.raw
4497.raw
4637.raw
10961.raw
2129.raw
5696.raw
3105.raw
10917.raw
4137.raw
2543.raw
5273.raw
11646.raw
11337.raw
798.raw
2027.raw
2260.raw
5386.raw
2918.raw
11983.raw
1324.raw
11651.raw
5680.raw
3838.raw
6347.raw
2551.raw
3775.raw
3557.raw
826.raw
1859.raw
6007.raw
11898.raw
1744.raw
830.raw
5965.raw
895.raw
3409.raw
5830.raw
5379.raw
6302.raw
235.raw
5814.raw
11524.raw
