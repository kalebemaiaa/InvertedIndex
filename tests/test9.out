47
231
408
468
520
638
821
825
1077
1153
1234
1346
1865
1976
2002
2142
2170
2219
2235
2303
2418
2635
2666
2679
2835
2866
2890
2918
3069
3100
3114
3209
3223
3225
3318
3343
3427
3531
3560
3609
3682
3911
3931
4182
4194
4286
4460
4481
4648
4720
5575
5598
5727
5868
5936
6256
6295
10781
10883
10993
11185
11221
11360
11541
11606
11744
